layouts/partials/language.html

```diff
@@ -33,7 +33,7 @@
                 <li>
                   <a
                     class="flex gdoc-language__entry"
-                    title="{{ i18n "language_switch_no_tranlation_prefix" }} {{ .LanguageName }}"
+                    title="{{ i18n "language_switch_no_translation_prefix" }} {{ .LanguageName }}"
```

Comment:
this change will break localizations. I could update them, but it's a coin toss as to whether that's desired or done by someone else.